include/openthread/platform/settings.h

```diff
@@ -80,10 +80,17 @@ enum
 /**
  * Performs any initialization for the settings subsystem, if necessary.
  *
- * @param[in]  aInstance The OpenThread instance structure.
+ * This function also sets the critical keys that should be stored in the secure area.
```

Comment:
Update the docs to use "sensitive" as well?

Reply:
Done.